ras-services.yml

```diff
@@ -193,6 +193,7 @@ services:
       - 8085:8085
     environment:
       - BACKSTAGE_API_URL=http://backstage:${BACKSTAGE_PORT}/backstage-api
+      - USE_UAA=0
```

Comment:
I can't find where this is an environmental variable in response_operations_ui

Reply:
My mistake, it's ras-backstage that needs this variable not response-operations-ui